rosdep/vcpkg.yaml

```diff
@@ -16,3 +16,27 @@ sdl2:
   windows:
     vcpkg:
       packages: [sdl2]
+librealsense2:
+  windows:
+    vcpkg:
+      packages: [realsense2]
+mavlink:
+  windows:
+    vcpkg:
+      packages: [mavlink]
```

Comment:
Is `mavlink` already on the Vcpkg upstream?

Reply:
Good catch - that is an unfinished project. Removed.